src/app/modules/item/pages/item-current-situation/item-current-situation.component.ts

```diff
@@ -92,8 +74,17 @@ export class ItemCurrentSituationComponent implements OnChanges {
     }));
   }
 
-  getHideSelection(): boolean {
-    return !this.watchedGroup && !!this.type && [ 'Task', 'Course' ].includes(this.type);
+  getHideSelection(type: ItemType): boolean {
+    return !this.watchedGroup && !!type && [ 'Task', 'Course' ].includes(type);
+  }
+
+  getLogItemCaption(type: ItemType): string {
+    return !this.watchedGroup && type === 'Chapter' || !!this.watchedGroup && [ 'Task', 'Course' ].includes(type)
+      ? $localize`History` : $localize`Log view`;
```

Comment:
It is "History" everywhere. The spec does not mention the use of `Log view` everywhere.

Reply:
Fixed